packages/glimmer-runtime/lib/scanner.ts

```diff
@@ -27,6 +28,13 @@ export default class Scanner {
     });
   }
 
+  scanInlineBlock(symbolTable: SymbolTable): InlineBlock {
```

Comment:
There's nothing really Partial specific? I feel like a proper factoring of the scanner would have a "scan this block relative to this symbol table". But whatever... call it `scanPartial` ;P